src/test/java/juja/sqlcmd/DatabaseManagerTest.java

```diff
@@ -33,4 +62,31 @@ public void connectWhenInvalidUserNameReturnsFalse() {
     public void connectWhenInvalidUserPasswordReturnsFalse() {
         assertFalse(databaseManager.connect("sqlcmd", "sqlcmd", "Notsqlcmd"));
     }
-}
+
+    @Test
+    public void getTableNamesWhenNoTablesInDbReturnsEmptyStringArray() throws SQLException {
+        String[] expected = new String[]{};
+        assertArrayEquals(expected, databaseManager.getTableNames());
+    }
+
+    @Test
+    public void getTableNamesWhenTwoTablesInDbReturnsStringArrayWithTableNames() throws SQLException {
+        executeSqlQuery("CREATE TABLE table1()");
+        executeSqlQuery("CREATE TABLE table2()");
+        String[] expected = new String[]{"table1", "table2"};
+        String[] actual = databaseManager.getTableNames();
+        Arrays.sort(actual);
+        assertArrayEquals(expected, actual);
+    }
+
+    private static void executeSqlQuery(String sqlQuery) throws SQLException {
+        try (Statement statement = connection.createStatement()) {
+            statement.execute(sqlQuery);
+        }
+    }
+
+    private static void dropAllTables() throws SQLException {
```

Comment:
Is that method safe? May be I can have another db in Postgres

Reply:
This queries have influence on database you have connection with, for other db in Postgres it's safe... I've check it already